File: pclassic/2016s/PClassic2016Stubs/stubs/JediAcademy.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;
import java.util.Arrays;

public class JediAcademy {
  //Fill out the body of this method
  public static int scoreSummary(int[] scores, int g){		
    return 0;
  }

  public static void main(String[] args) throws IOException {
		
    BufferedReader b = new BufferedReader(new FileReader("JediAcademyIN.txt"));

    while(b.ready()){			
      String in = b.readLine();
      String[] inBreakLast = in.split("--");
			
      int last = Integer.parseInt(inBreakLast[1]);
			
      String[] inArr = inBreakLast[0].split(",");
      int[] scores = new int[inArr.length];
			
      for (int i = 0; i < inArr.length; i++){
        scores[i] = Integer.parseInt(inArr[i]);
      }
			
      System.out.println(scoreSummary(scores, last));
    }
		
    b.close();
				
  }

}

File: pclassic/2016s/PClassic2016Stubs/stubs/JediAcademy.py
# Change the body of this method
def best_grouping(scores, G):
    return 0

if __name__ == "__main__":
    with open("JediAcademyIN.txt", "r") as f:
        while True:
            s = f.readline()
            if s == "":
                break
            data = s.split("--")
            scores = [int(x) for x in data[0].split(",")]
            G = int(data[1])
            print(best_grouping(scores, G))
